src/main/java/mate/academy/springboot/datajpa/repository/CategoryRepository.java

```diff
@@ -0,0 +1,14 @@
+package mate.academy.springboot.datajpa.repository;
+
+import javax.transaction.Transactional;
+import mate.academy.springboot.datajpa.model.Category;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Modifying;
+import org.springframework.data.jpa.repository.Query;
+
+public interface CategoryRepository extends JpaRepository<Category, Long> {
+    @Transactional
+    @Modifying
+    @Query("update Category set name =?1 where id = ?2")
+    void update(String name, Long id);
```

Comment:
```suggestion
```
You can remove this using the update that JpaRepository provides by default. Also you don't need two parameters in the update just the Category